app/services/flood_risk_engine/reference_number.rb

```diff
@@ -0,0 +1,58 @@
+module FloodRiskEngine
```

Comment:
this class makes it hard to follow whats actually being built and returned. perhaps over engineered giving its really just formatting a string